assets/js/modules/analytics-4/datastore/accounts.js

```diff
@@ -43,24 +44,41 @@ import { createFetchStore } from '../../../googlesitekit/data/create-fetch-store
 import { actions as errorStoreActions } from '../../../googlesitekit/data/create-error-store';
 import { createValidatedAction } from '../../../googlesitekit/data/utils';
 import { isValidAccountSelection } from '../utils/validation';
+import { caseInsensitiveListSort } from '../../../util/case-insensitive-sort';
+import {
+	populateAccountID,
+	populatePropertyAndAccountIds,
+} from '../utils/account';
 
 const { receiveError, clearError, clearErrors } = errorStoreActions;
 
 const fetchGetAccountSummariesStore = createFetchStore( {
 	baseName: 'getAccountSummaries',
-	controlCallback() {
+	controlCallback( { pageToken } ) {
 		return API.get(
 			'modules',
 			'analytics-4',
 			'account-summaries',
-			{},
+			pageToken,
```

Comment:
This also needs to be an object – it's currently just passing through the string, which is why this is being split into an object of letters in my last comment.

```suggestion
			{ pageToken },
```